### PR TITLE
linux: rockchip: fix missing cpufreq on ROCK 5B

### PR DESCRIPTION
**Pull request by RadxaNaoki**

### CODE REVIEW

File: linux/.common/kconfig.conf [hunk 1]
```diff
@@ -67,14 +67,14 @@ CONFIG_SPI_BITBANG=m
 CONFIG_SPI_GPIO=m
 CONFIG_SPI_MESON_SPICC=m
 CONFIG_SPI_MESON_SPIFC=m
-CONFIG_SPI_ROCKCHIP=m
+CONFIG_SPI_ROCKCHIP=y
```
Comment (RadxaYuntian) [inline linux/.common/kconfig.conf:70]: Why this needs to be built-in as well?

File: linux/.common/kconfig.conf [hunk 2]
```diff
@@ -67,14 +67,14 @@ CONFIG_SPI_BITBANG=m
...
 CONFIG_SPI_ROCKCHIP_SFC=m
 
 # Fix panfrost cooling device error
 # https://www.mail-archive.com/linux-kernel@vger.kernel.org/msg2153527.html
 CONFIG_DEVFREQ_THERMAL=y
 CONFIG_AMLOGIC_THERMAL=m
-CONFIG_ROCKCHIP_THERMAL=m
+CONFIG_ROCKCHIP_THERMAL=y
```
Comment (RadxaYuntian) [inline linux/.common/kconfig.conf:77]: Have you tried to modify `/etc/modules` to early loading this driver? I tried to keep them as modules so user can disable unneeded features at will: https://github.com/radxa-repo/rbuild/blob/main/common/overlays/common/etc/modules

File: linux/.common/kconfig.conf (hunk 1 above)
Comment (RadxaNaoki) [reply linux/.common/kconfig.conf:70]: no idea for now

File: linux/.common/kconfig.conf (hunk 2 above)
Comment (RadxaNaoki) [reply linux/.common/kconfig.conf:77]: /etc/modules doesn't fix this issue

File: linux/.common/kconfig.conf (hunk 1 above)
Comment (RadxaYuntian) [reply linux/.common/kconfig.conf:70]: What I mean is if you don't include this change will cpufreq work?

Comment (RadxaNaoki) [reply linux/.common/kconfig.conf:70]: cpufreq doesn't work if CONFIG_SPI_ROCKCHIP is "m"

Comment (RadxaYuntian) [reply linux/.common/kconfig.conf:70]: In that case let's add those 2 lines in `linux/rockchip/kconfig.conf` with appropriate comment noting this is for ROCK 5B cpufreq. That file is applied after the `linux/rockchip/0001-common/kconfig.conf`, and we can keep those as module for upstream kernel.

Comment (RadxaNaoki) [reply linux/.common/kconfig.conf:70]: sounds good. I'll make new patch.

**Pull request merged**